microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -22,89 +22,105 @@ ms.date: 05/04/2021
-
-This guide assumes that you are familiar with the basics of [web development](https://developer.mozilla.org/docs/Learn) and [Microsoft Edge DevTools](../../devtools-guide-chromium/overview.md).
+You can view the resources that make up a web page from within several tools (or _panels_) in Microsoft Edge DevTools, including:
+*  Network tool
+*  Sources tool
+*  Application tool
 
+See also:
+*  [Learn web development](https://developer.mozilla.org/docs/Learn) at Mozilla.org
+*  [Overview of DevTools](../../devtools-guide-chromium/overview.md)
 
+To open DevTools, right-click the webpage, and then select **Inspect**. Or press `Ctrl`+ `Shift`+ `I` (Windows, Linux) or `Command`+ `Option`+ `I` (macOS).
 <!-- ====================================================================== -->
 ## Open resources from the Command Menu
 
-When you know the name of the resource that you want to inspect, the **Command Menu** provides a fast way of opening the resource.
+When you know the name of the resource that you want to inspect, the **Command Menu** in DevTools provides a fast way of finding and opening the resource.
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**. Or press `Ctrl`+ `Shift`+ `I` (Windows, Linux) or `Command`+ `Option`+ `I` (macOS).
+
+1. When DevTools has focus, press `Ctrl`+ `P` (Windows, Linux) or `Command`+`P` (macOS). Or, you can click the (![Customize icon](../media/customize-devtools-icon-light-theme.png)) menu button in DevTools and then select **Open file**.  The Command Menu opens, displaying the **Open File** list:
 
-1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS).  The **Open File** dialog opens.
+   ![The Command Menu, displaying the Open File list](../media/resources-command-menu-empty.msft.png)
 
-   ![The Open File dialog.](../media/resources-command-menu-empty.msft.png)
+1. Start typing the filename and press `Enter` when the correct file is highlighted in the autocomplete box or select the file from the dropdown list.
 
-1. Select the file from the dropdown list, or start typing the filename and press `Enter` once the correct file is highlighted in the autocomplete box.
+   Type filename:
 
    ![Type a filename in the Open File dialog.](../media/resources-command-menu-file-search.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Open resources in the Network tool
 
-See [Inspect the details of a resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
-<!--why?-->
+In the **Network** tool, you can inspect the resources that make up the current webpage, such as `.html`, `.css`, `.js`, and image files.  To learn how to get the details for a specific resource, see [Inspect the details of the resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
 
-<!-- need lead-in -->
+Show network resource details:
```

Comment:
```suggestion
When you go to the **Network** tool and then refresh the page, all the webpage's resource files are listed in the **Network** tool, one row per resource file:

![Webpage resource files listed in the Network Log, one resource file per row](../media/resources-network-resources.msft.png)

When **All** is selected as the resources filter, all the retrieved resource files for the current webpage are listed, in the lower left:

![Inspect a resource in the Network tool](../media/resources-network-response.msft.png)

The above screenshot shows `getstarted.html`, though a folder is selected in the lower left.

To learn how to show the network activity that's generated by the page resources, see [Log network activity](../network/index.md#log-network-activity) in the article  _Inspect network activity_.
```
here's the remainder of the redesigned h2 section, including a below section moved up into here.